lunch-go-where/src/pages/Landing.js

```diff
@@ -164,6 +191,15 @@ const Landing = () => {
               />
             </FormControl>
             <br />
+            {showErrorLabel ? (
+              <label>
+                Password is not valid. Please include a letter and a number
```

Comment:
What happens if a user tries to type a space?

Reply:
Will work on this after card component is done. :D